docs/getting-started/windows-manual.md

```diff
@@ -72,3 +84,36 @@ Start-Service splunk-otel-collector
 ```
 
 The collector logs and errors can be viewed in the Windows Event Viewer.
+
+### Docker
+
+Create build of otel collector on local system.
+```bash
+$ git clone https://github.com/signalfx/splunk-otel-collector.git
+$ cd splunk-otel-collector
+$ docker build -t otelcol --build-arg SMART_AGENT_RELEASE=5.11.2 -f .\cmd\otelcol\Dockerfile.windows .\cmd\otelcol\
+```
+
+Deploy the latest Docker image.
+
+```bash
+$ docker run --rm -e SPLUNK_ACCESS_TOKEN=12345 -e SPLUNK_REALM=us0  `
+	-p 13133:13133 -p 14250:14250 -p 14268:14268 -p 4317:4317 -p 6060:6060  `
+	-p 8888:8888 -p 9080:9080 -p 9411:9411 -p 9943:9943 --name=otelcol quay.io/signalfx/splunk-otel-collector-windows:latest
+```
+### Custom Configuration
+
+When we want to make changes in to the default configuration YAML file, for that create a
+custom configuration YAML file. Then after we can use `SPLUNK_CONFIG` environment variable  or
+command line argument `--config` to provide the path to this file.
+
+For example in Docker:
```

Comment:
```suggestion
Example with `SPLUNK_CONFIG`:
```

Also, let's add another example for the `--config` option.

Reply:
Example for the `--config` option added in the documentation.